src/shelly_main.cpp

```diff
@@ -822,6 +825,11 @@ extern "C" bool mgos_ota_merge_fs_should_copy_file(const char *old_fs_path,
       "relaydata",
       "index.html",
       "conf9_backup.json",
+      "conf3.json",
+      "hwinfo_struct.json",
```

Comment:
actually, hwinfo_struct is very important. without it, stock firmware won't even run, so definitely need to keep it.
let's keep the rest of them too. i know conf3 messages are a bit annoying but i believe the file may have important factory settings that we don't want to wipe.

Reply:
Oh, I expected that the `hwinfo_struct` was already reverted.

Reply:
I observed some crashes with the existing of `conf3.json`

Reply:
would be interesting to know which exact settings cause it. let's do it separately.

Reply:
It was something with color mode, that was the only key in conf3